Jigsaw-app/Jigsaw/Info.plist

```diff
@@ -28,7 +28,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<string>210210</string>
```

Comment:
We may revert this change and get rid of the build phase that updates the project version for now.